sys/net/gnrc/routing/ipv6_auto_subnets/gnrc_ipv6_auto_subnets.c

```diff
@@ -302,8 +304,8 @@ static void _configure_subnets(uint8_t subnets, uint8_t start_idx, gnrc_netif_t
         _remove_old_prefix(downstream, &new_prefix, new_prefix_len, &ext_opts);
 
         /* configure subnet on downstream interface */
-        gnrc_netif_ipv6_add_prefix(downstream, &new_prefix, new_prefix_len,
-                                   valid_ltime, pref_ltime);
+        idx = gnrc_netif_ipv6_add_prefix(downstream, &new_prefix, new_prefix_len,
+                                         valid_ltime, pref_ltime);
```

Comment:
Unrelated, as this function already seems to have this return value, but I am not quite sure about the semantics of the return value: The name of the function implies that the function adds a prefix, but it returns the index to an address of that interface. Which address? Depending on the length, a prefix can result in many possible addresses. Maybe this can just be clarified in the doc (which I could not find)? I assume it is the address that results when prepending the prefix to the link layer address based IIDs